src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectIT.java

```diff
@@ -126,10 +126,14 @@ private static void dropAll(final DatabaseObject... databaseObjects) {
         }
     }
 
-    private static Connection getHiveConnection() throws ClassNotFoundException, SQLException {
-        Class.forName("org.apache.hive.jdbc.HiveDriver");
-        return DriverManager.getConnection("jdbc:hive2://localhost:" + HIVE_EXPOSED_PORT + "/" + SCHEMA_HIVE,
-                HIVE_USERNAME, HIVE_PASSWORD);
+    private static Connection getHiveConnection() throws ClassNotFoundException, SQLException, MalformedURLException,
+            IllegalAccessException, InstantiationException, NoSuchMethodException, InvocationTargetException {
+        final File file = new File("src/test/resources/integration/driver/hive/HiveJDBC41.jar");
+        final URLClassLoader urlClassLoader = new URLClassLoader(new URL[] { file.toURI().toURL() },
+                HiveSqlDialectIT.class.getClassLoader());
+        final Class<?> driverClass = urlClassLoader.loadClass("com.cloudera.hive.jdbc41.HS2Driver");
```

Comment:
Can we please extract the loading of the JDBC driver to dedicated method, like `loadJDBCDriver` or similar?